### PR TITLE
sorted features and indices for convenience

### PR DESCRIPTION
**Pull request by colehurwitz**
Added indices to multisortingextractor and sorted indices/features for convenience.

### CODE REVIEW

**Comment by colehurwitz** [comment]
@alejoe91

**Comment by alejoe91** [comment]
I'll look into it and merge tomorrow

**Comment by colehurwitz** [comment]
Great. Once this is merged, we can look into merging the spiketoolkit PRs. I think this will help the split_units function in the other PR.

File: spikeextractors/sortingextractor.py [hunk 1]
```diff
@@ -144,8 +148,12 @@ def set_unit_spike_features(self, unit_id, feature_name, value, indexes=None):
                             raise ValueError("feature values should have the same length as the spike train")
                 else:
                     if isinstance(feature_name, str) and len(value) == len(indexes):
-                        self._features[unit_id][feature_name] = value
-                        self._features[unit_id][feature_name + '_idxs'] = np.array(indexes)
+                        indexes = np.array(indexes)
+                        indexes_sorted_indices = np.argsort(indexes)
```
Comment (alejoe91) [inline spikeextractors/sortingextractor.py:152]: this shoudl be sort! They are indexes already

**Pull request merged**